samples/net/ocpp/src/main.c

```diff
@@ -298,14 +303,22 @@ int main(void)
 
 	wait_for_network();
 
+	if (strlen(CONFIG_NET_SAMPLE_OCPP_SERVER) < 1) {
+		LOG_ERR("CONFIG_NET_SAMPLE_OCPP_SERVER is empty. Please set it in prj.conf");
+		return -EINVAL;
+	}
+
 	ret = ocpp_getaddrinfo(CONFIG_NET_SAMPLE_OCPP_SERVER, CONFIG_NET_SAMPLE_OCPP_PORT, &ip);
 	if (ret < 0) {
 		return ret;
 	}
 
 	csi.cs_ip = ip;
 
-	ocpp_get_time_from_sntp();
+	ret = ocpp_get_time_from_sntp();
+	if (ret < 0) {
```

Comment:
should not return even sntp unreachable.. e.g system time is maintained in RTC or some other source.